algo/src/main/java/com/hartwig/serve/sources/ckb/ActionableEvidenceFactory.java

```diff
@@ -47,12 +49,13 @@ class ActionableEvidenceFactory implements ActionableEntryFactory {
         RESISTANT_DIRECTIONS.add("resistant");
         PREDICTED_RESISTANT_DIRECTIONS.add("predicted - resistant");
 
+        NO_BENEFIT_DIRECTIONS.add("no benefit");
+        DECREASED_RESPONSE_DIRECTIONS.add("decreased response");
+
         DIRECTIONS_TO_IGNORE.add("unknown");
         DIRECTIONS_TO_IGNORE.add("not applicable");
         DIRECTIONS_TO_IGNORE.add("conflicting");
-        DIRECTIONS_TO_IGNORE.add("no benefit");
         DIRECTIONS_TO_IGNORE.add("not predictive");
```

Comment:
Did you check if "not predictive" may be mappable to `NO_BENEFIT`?

Reply:
I looked at the documents in the weekly dump, and "not predictive" is not in there, so overlooked it but the definition is on CKB's website (I think "not predictive" is quite new, the docs in the weekly dump are from 2019..). Looking at the definition: _"Evidence from peer-reviewed published literature from one or more studies demonstrating that a molecular profile does not confer sensitivity or resistance to a specific monotherapy or combination therapy"_, I think we could indeed map "not predictive" to `NO_BENEFIT`.

Reply:
There are actually >100 entries (see also `select * from actionability where responseType = 'not predictive';` in `ckb_latest`). I've scanned a bunch of them and they feel quite similar to `NO_BENEFIT`. Sounds like a good improvement to map these then! I will double-check with Sara nonetheless